### ISSUE TITLE
PlaceAtLocation never unsubscribes from OnLocationUpdatedEvent/OnProviderRestartEvent

### ISSUE THREAD

**Issue by TobiasWehrum** [dmbfm/unity-ar-gps-location-issues#10]
PlaceAtLocation never unsubscribes from OnLocationUpdatedEvent/OnProviderRestartEvent (and maybe even other events?), which is a problem when it gets destroyed. (And there isn't even a way to unsubscribe from those events at the moment.)

**Comment by dmbfm** [comment]
You are correct, we just added a fix for that which should be out soon, thanks a lot for pointing it out!
